contributors.yml

```diff
@@ -63,4 +63,6 @@
 - turansky
 - underager
 - vijaypushkin
+- rtmann
```

Comment:
^^ I don't think this is intentional - potentially leftover from the merge conflict resolution?

Reply:
Sorry. It got sorted not added.